src/current/v23.2/ui-network-latency-page.md

```diff
@@ -33,15 +38,37 @@ For instance, the cluster shown above has nodes in `us-west1`, `us-east1`, and `
 
 ### No connections
 
-Nodes that have lost a connection are displayed in a separate color. This can help you locate a network partition in your cluster.
+Nodes that have completely lost connectivity are color-coded depending on connection status:
+
+- Orange with an exclamation mark indicates `unknown connection state`.
+- Red with a [no symbol](https://www.wikipedia.org/wiki/No_symbol) indicates `Failed connection`. 
+
+This information can help you diagnose a network partition in your cluster.
+
+<img src="{{ 'images/v23.2/ui_network_latency_matrix_suspect_node.png' | relative_url }}" alt="DB Console Network Latency suspect node" style="border:1px solid #eee;max-width:100%" />
+
+Hover over a cell to display more details:
+
+- The direction of the connection marked by `From` and `To`.
+- Locality metadata for origin and destination.
+- Connection status.
+- The error message that resulted from the most recent connection attempt.
+
+This specific information can help you understand the root cause of the connectivity issue.
 
 {{site.data.alerts.callout_info}}
 A network partition prevents nodes from communicating with each other in one or both directions. This can be due to a configuration problem with the network, such as when allowlisted IP addresses or hostnames change after a node is torn down and rebuilt. In a symmetric partition, node communication is broken in both directions. In an asymmetric partition, node communication works in one direction but not the other.
 
 The effect of a network partition depends on which nodes are partitioned, where the ranges are located, and to a large extent, whether [localities]({% link {{ page.version.version }}/cockroach-start.md %}#locality) are defined. If localities are not defined, a partition that cuts off at least (n-1)/2 nodes will cause data unavailability.
 {{site.data.alerts.end}}
 
-Click the **NO CONNECTIONS** link to see lost connections between nodes or [localities]({% link {{ page.version.version }}/cockroach-start.md %}#locality), if any are defined.
+### Node liveness status
+
+Hover over a node's ID in the row and column headers to show the node's liveness status, such as `healthy` or `suspect`. Node liveness status is also indicated by the colored circle next to the Node ID: green for `healthy` or red for `suspect`.
+
+If a `suspect` node stays offline for the duration set by[`server.time_until_store_dead`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-server-time-until-store-dead) (5 minutes by default), the [cluster considers the node "dead"]({% link {{ page.version.version }}/node-shutdown.md %}#process-termination) and the node is removed from the matrix.
+
+The number of `LIVE` (healthy), `SUSPECT`, `DRAINING` and `DEAD` nodes is displayed in the [Cluster Overview page]({% link {{ page.version.version }}/ui-cluster-overview-page.md %}).
```

Comment:
Also decommissioning? Did I see something about that somewhere?

Reply:
Decommissioning is displayed in the Status column of the Node List on the Cluster Overview page, under Node Status on the top panel it is not displayed:
<img width="328" alt="Screenshot 2023-09-14 at 10 11 46 AM" src="https://github.com/cockroachdb/docs/assets/58752716/1f1ace2d-db49-40ee-bbd1-4133a32cb83d">

I qualified the sentence with "under Node Status".